build_tools/python/e2e_model_tests/test_definitions.py

```diff
@@ -63,54 +63,58 @@ class ModelTestConfig(object):
 
 TEST_CONFIGS = [
     # mobilenet_v1_fp32_correctness_test
-    ModelTestConfig(
-        name="mobilenet_v1_fp32_correctness_test",
-        imported_model=iree_definitions.ImportedModel.from_model(
-            tflite_models.MOBILENET_V1
-        ),
-        execution_config=module_execution_configs.ELF_LOCAL_SYNC_CONFIG,
-        expected_output="mobilenet_v1_fp32_expected_output.txt",
-        unsupported_platforms=[
-            CMakePlatform.LINUX_RISCV32,
-            CMakePlatform.ANDROID_ARMV8_A,
-        ],
-    ),
+    # TODO(#14775): Re-enable the tests.
```

Comment:
@hcindyl @dcaballe fyi

Reply:
Can set `XFAIL_PLATFORMS all`

https://github.com/openxla/iree/blob/d81e29b6993b63aed2421dfd5d2ed05663456d52/build_tools/cmake/iree_run_module_test.cmake#L32

instead of commenting them out.

Reply:
I can set `xfail_platforms`, but that would need to list all the platforms. There is no `all` in the enum. I think commenting them out is okay.